baseapp/abci_test.go

```diff
@@ -1315,6 +1369,49 @@ func TestABCI_GetBlockRetentionHeight(t *testing.T) {
 	}
 }
 
+// Verifies that the Commiter is called with the checkState.
+func TestCommiterCalledWithCheckState(t *testing.T) {
+	t.Parallel()
+
+	logger := defaultLogger()
+	db := dbm.NewMemDB()
+	name := t.Name()
+	app := baseapp.NewBaseApp(name, logger, db, nil)
+
+	wasCommiterCalled := false
+	app.SetCommiter(func(ctx sdk.Context) {
```

Comment:
These tests were moved by the Cosmos team to be in a different package, so we can no longer reference `app.checkState` and `app.deliverState` directly. Instead, I am checking `IsCheckTx()` on the context.